projects/exui-common-lib/src/lib/components/find-service/find-service.component.spec.ts

```diff
@@ -1,28 +1,159 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { ReactiveFormsModule } from '@angular/forms';
+import { FormArray, FormControl, FormGroup, FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { MatAutocompleteModule, MatOptionModule } from '@angular/material';
 import { SearchServiceComponent } from '../search-service/search-service.component';
 import { FindServiceComponent } from './find-service.component';
 
 describe('FindServiceComponent', () => {
   let component: FindServiceComponent;
   let fixture: ComponentFixture<FindServiceComponent>;
+  let service = {
+    key: '01',
+    label: '01-label',
+  };
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      imports: [ReactiveFormsModule, MatAutocompleteModule, MatOptionModule],
+      imports: [ReactiveFormsModule, MatAutocompleteModule, MatOptionModule, FormsModule],
       declarations: [FindServiceComponent, SearchServiceComponent],
-      providers: []
-    })
-    .compileComponents();
+      providers: [],
+    }).compileComponents();
   }));
 
   beforeEach(() => {
     fixture = TestBed.createComponent(FindServiceComponent);
     component = fixture.componentInstance;
+    component.field = {
+      type: 'find-service',
+      name: 'services',
+      options: [],
+      minSelected: 1,
+      maxSelected: null
+    };
+    component.form = new FormGroup({
+      services: new FormArray([]),
+    });
+    let services = [ {
+      key: '01',
+      label: '01-label',
+    },
+    {
+      key: '03',
+      label: '03-label',
+    },
+    {
+      key: null,
+      label: '02-label',
+    },
+  ]
+    component.services = services;
```

Comment:
I think component.services shouldn't be set directly? it should be set from component.field.options

Reply:
This is spec to setting its mock values etc